nx.json

```diff
@@ -47,5 +39,10 @@
       "style": "scss"
     }
   },
-  "defaultProject": "lumberjack-app"
+  "defaultProject": "lumberjack-app",
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"]
```

Comment:
question: What does the caret mean?

Reply:
This comes from the migration, which I didn't notice until now. I'll take a look at the docs.

Reply:
https://nx.dev/configuration/projectjson#dependson

Reply:
With the caret it is telling that it needs its dependencies to be built before it can run that target.

Reply:
We already have this setting in the global Nx configuration: https://github.com/ngworker/lumberjack-next/blob/91a541aeec5368b930704e28e197a6da8d74c42b/nx.json#L27-L34